react-app/constants.js

```diff
@@ -1,18 +1,9 @@
-export const basicFeatures = ['View a list of folders and files', 'Navigate through folders', 'Download files', 'Preview files', 'Search files by name', 'Select files and/or folders'];
+export const basicDescriptionInstructions = 'Use this component to upload and organize files that students can access directly in the unit.';
```

Comment:
This description is visible from the LMS, but students cannot do file uploads. I think we should modify that description.

Reply:
I've changed it to this

![lms-view](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/f621c496-669c-47ed-9181-9feeb7d7b458)